libs/cardano-ledger-test/cardano-ledger-test.cabal

```diff
@@ -46,7 +46,13 @@ library
     Test.Cardano.Ledger.Alonzo.Tools
     Test.Cardano.Ledger.BaseTypes
     Test.Cardano.Ledger.Examples.BabbageFeatures
-    Test.Cardano.Ledger.Examples.TwoPhaseValidation
+    Test.Cardano.Ledger.Examples.AlonzoUTXOW
+    Test.Cardano.Ledger.Examples.AlonzoUTXOWValidTx
+    Test.Cardano.Ledger.Examples.AlonzoUTXOWInvalidTx
```

Comment:
Minor optional suggestion: I think it would read a little easier this way:
```suggestion
    Test.Cardano.Ledger.Examples.AlonzoValidTxUTXOW
    Test.Cardano.Ledger.Examples.AlonzoInvalidTxUTXOW
```

Reply:
I agree,  your suggestions seem more pleasant for some reason, will change